improver/standardise.py

```diff
@@ -122,6 +123,40 @@ def _remove_scalar_coords(cube: Cube, coords_to_remove: List[str]) -> None:
             except CoordinateNotFoundError:
                 continue
 
+    @staticmethod
+    def _modify_scalar_coord_value(
+        cube: Cube, coord_modification: Dict[str, float]
+    ) -> None:
+        """Modifies the value of each specified scalar coord (dictionary key)
+        to the provided value (dictionary value). Note that data types are not
+        enforced here as the subsequent enforcement step will fulfil this
+        requirement. Units are assumed to be the same as the original
+        coordinate value. Modifying multi-valued coordinates or time
+        coordinates is specifically prevented as there is greater scope to
+        harm data integrity (i.e. the description of the data and the data
+        becoming misaligned).
+
+        If the coordinate does not exist the modification request is silently
+        skipped.
+
+        Args:
+            cube:
+                Cube to be updated in place
+            coord_modification:
+                Dictionary defining the coordinates (keys) to be modified
+                and the values (values) to which they should be set.
+        """
+        for coord, value in coord_modification.items():
+            if cube.coords(coord):
+                if cube.coords(coord, dim_coords=True):
+                    raise ValueError(
+                        "Modifying dimension coordinate values it not allowed "
+                        "due to the risk of introducing errors."
+                    )
+                if _is_time_coord(cube.coord(coord)):
+                    raise ValueError("Modifying time coordinates it not allowed.")
+                cube.coord(coord).points = np.array([value])
```

Comment:
At the moment, I think this allows a multi-valued `aux_coord` to be changed by this method, is this intended? If not, the length of `value` could also be checked an raise an exception if not 1.